packages/next/README.md

```diff
@@ -251,9 +251,9 @@ function MyImage() {
 export default MyImage
 ```
 
-To serve static files from the root directory you can add a folder called `public` and reference those files from the root, e.g: `/robots.txt`
+To serve static files from the root directory you can add a folder called `public` and reference those files from the root, e.g: `/robots.txt`.
 
-_Note: Don't name the `static` or `public` directory anything else. The names can't be changed and are the only directories that Next.js uses for serving static assets_
+_Note: Don't name the `static` or `public` directory anything else. The names can't be changed and are the only directories that Next.js uses for serving static assets._
```

Comment:
Should it be like this instead?
```suggestion
_Note: Don't name the `static` or `public` directory anything else. The names can't be changed and are the only directories that Next.js uses for serving static assets_.
```

Reply:
`_` is there to make the text italic, it isn't visible, and the dot is part of the text.